models /articles.model.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../db/connection");
+
+exports.requireArticles = () => {
+  return db.query("SELECT * FROM articles").then((body) => {
+    return body.rows;
+  });
+};
+
+exports.requireArticleById = (article_id) => {
+
+
+  return db
+    .query(
+      `SELECT * FROM articles 
+  WHERE article_id = ${article_id};`
+    )
+    .then((body) => {
+      return body.rows
```

Comment:
What is body.rows? If we only want one item, shouldn't we take this out of the rows array here?